homework_8/views/v1/categories.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework.views import APIView
+from rest_framework.generics import get_object_or_404
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework import status
+
+from homework_8.models import Category
+from homework_8.serializers.categories import CategoryCreateSerializer
+
+
+class CategoryListCreateView(APIView):
```

Comment:
If the class is intended only for creating a new object - it is better to rename it. At the moment it is assumed by the name that the class will work for displaying the list of objects and for creating a new object